website/content/docs/ecs/get-started/requirements.mdx

```diff
@@ -7,16 +7,10 @@ description: >-
 
 # Requirements
 
-Currently, the following requirements must be met in order to install Consul on ECS:
+The following requirements must be met in order to install Consul on ECS:
 
 1. **Terraform:** The tasks that you want to add to the service mesh must first be modeled in Terraform.
-1. **Launch Type:** Only the Fargate launch type is currently supported.
+1. **Launch Type:** Fargate and EC2 launch types are supported.
 1. **Subnets:** ECS Tasks can run in private or public subnets. Tasks must have [network access](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-error-ecr/) to Amazon ECR to pull images.
```

Comment:
```suggestion
1. **Subnets:** ECS Tasks can run in private or public subnets. Tasks must have [network access](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-error-ecr/) to Amazon ECR or other public container registries to pull images.
```
These tasks can pull images from other container registries.

Reply:
👍 

Clarifying: This called out ECR in particular because that is where we publish our `consul-ecs` image - https://gallery.ecr.aws/hashicorp/consul-ecs for people to consume on AWS.